content/en/Platform Deep Dive/Scans/target_auth.md

```diff
@@ -12,9 +12,10 @@ If your website has areas that require authentication, you may provide the DAST
 
 When a scan is started and the target has a login configuration, the first thing the crawler does is log into the application (web target) to obtain a session. Upon successful login, it will start crawling the app. While the crawler is running, it constantly verifies whether the session is still valid. Currently, this check is performed automatically based on the login configuration, but soon we will have the option to configure how the loss of session can be detected.
 
+
 # Basic steps
 
-To add authentication, toggle on the “Authenticated Scan” option and then:
+To add authentication using a simple login form, go to the target's Advanced Settings, toggle on the “Login form” option and then:
```

Comment:
Since now we have to select one of the options and they are described in length later, I think we could just leave a brief sentence about going to advaned settings, since both options are described in detail later

Reply:
> I think we could just leave a brief sentence about going to advanced settings,

I did't understand this part.

Reply:
I meant that we can leave something short in `Basic Steps` and leave the rest of the description for the specific option (Sequence Recorder or Login Form).

> ...this check is performed automatically based on the login configuration, but soon we will have the option to configure how the loss of session can be detected.

And then add...

> To enable authentication, you can go to the target's Advanced Settings and select either `Login Form` or `Login Sequence`.

And that's it. The rest can be in `Using Login Sequence` or `Using a Login Form`

Reply:
I thought the basic steps were about a simple happy scenario. In this case, a simple login form. That's why I kept it.
Otherwise, it can be removed alltogether.

Reply:
@mtsuji-cobalt @eugenerevzin what do you think of this Basic Steps section?

Reply:
I agree with David - I think Basic Steps can be generally describing how to deal with your Authentication settings in Advanced Settings: none, login form, login sequence, with the latter two linking to the section for that login method within the page.

Reply:
Thanks for the feedback. I simplified the section and moved the other sentence to the form login section: https://deploy-preview-487--cobalt-docs.netlify.app/platform-deep-dive/scans/target_auth